fixtures/ezevent/vip-r.json

```diff
@@ -0,0 +1,101 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "VIP-R+",
+  "shortName": "VIP-R+",
+  "categories": ["Stand", "Color Changer", "Strobe"],
+  "meta": {
+    "authors": ["Herve Pequignet"],
+    "createDate": "2024-02-08",
+    "lastModifyDate": "2024-02-08"
+  },
+  "links": {
+    "manual": [
+      "https://ezevent.net"
+    ],
+    "productPage": [
+      "https://ezevent.net/vip-r-plus/"
+    ]
+  },
+  "rdm": {
+    "modelId": 1284
+  },
+  "physical": {
```

Comment:
The QLC+ exporter is throwing an error about physical dimensions. I suspect they probably need adding to silence that.

Reply:
Thanks Peternewman. No need the dimension of the product